conf/messages.cy

```diff
@@ -0,0 +1,82 @@
+common.service-name = Cyfrif Treth Busnes
+common.fullStop = .
+common.pageTitle = {0} - {1} - GOV.UK
+common.finish = Gorffen
+common.confirmAndContinue = Cadarnhau ac yn eich blaen
+common.errorSummary.heading = Mae problem wedi codi
+common.continue = Yn eich blaen
+
+base.back = Yn ôl
+
+featureSwitch.title = Feature switches
+featureSwitch.heading = Nodweddion
+featureSwitch.submit = Cyflwyno
+
+common.error.prefixTitle = Gwall: {0}
+error.boolean = Please give an answer
+error.required = Please enter a value
+error.summary.title = Mae problem wedi codi
+
+navTitle.nonAgent = Cyfrif Treth Busnes
+navTitle.agent = Manylion TAW eich cleient
+navTitle.vat = TAW
+
+global.error.InternalServerError500.title = Mae problem gyda’r gwasanaeth – TAW – GOV.UK
+global.error.InternalServerError500.heading = Mae’n ddrwg gennym – mae problem gyda’r gwasanaeth
+global.error.InternalServerError500.message = Rhowch gynnig arall arni yn nes ymlaen.
+
+unauthorised.agent.title = Ni allwch ddefnyddio’r gwasanaeth hwn eto
+unauthorised.agent.useThisService = I ddefnyddio’r gwasanaeth hwn, bydd angen i chi
+unauthorised.agent.setUpAccount = agor cyfrif gwasanaethau asiantau
+
+unauthorised_non_agent.title = Ni allwch ddefnyddio’r gwasanaeth hwn eto
+unauthorised_non_agent.heading = Ni allwch ddefnyddio’r gwasanaeth hwn eto
+unauthorised_non_agent.text = Mae’n rhaid i chi
+unauthorised_non_agent.link = gofrestru i ddefnyddio meddalwedd er mwyn cyflwyno’ch Ffurflenni TAW.
+
+session_expired.title = For your security, this service has been reset
+session_expired.heading = For your security, this service has been reset
+session_expired.guidance = The details you have given have been deleted because you did not continue the service for 15 minutes.
```

Comment:
These messages don't seem to be used in other project other than vat-designatory-details-frontend and in that project they also have no welsh transltion 
They are also not used in any of the views so can be removed

Reply:
Removed